src/Sarif.Driver/Sdk/PluginDriverCommand.cs

```diff
@@ -122,6 +129,11 @@ internal bool ValidateOutputFileCanBeCreated(IAnalysisContext context, string ou
             return succeeded;
         }
 
+        internal bool ValidateFileSizeInKilobytes(int fileSizeInKilobytes)
```

Comment:
>ValidateFileSizeInKilobytes [](http://example.com/codeflow?start=22&length=27)

This factoring looks questionable to be. Your method name is as long as its implementation. :) 

In general, a utility method only has value if it contains some specialized logic that you don't want to maintain in multiple places. A simple 'greater than 0' comparison doesn't seem to meet that test.

There is another subtle issue 'ValidateFileSizeInKilobytes' is suggestive, it suggests more validation is done. You encode 'kilobytes' in the name and 'file size' and yet the implementation of this helper seems to be 'VerifyIsGreaterThanZero', and that's it. 

Having said all this, I don't even see a caller for this method. :) So maybe I'm making a whole lot of pointing out you left some intermediate code in the file. :)

Final point, though, just to keep beating this to death, this helper doesn't access instance data and, if it were to survive, could be static.